### DISCUSSION TITLE
Gode eksempler

### DISCUSSION THREAD

**Discussion by kristineSorensen** [fremtind/jokul#2387] [Annet]
I Interaksjonsdesignforum (IXD forum ) i dag har vi diskutert skjemaer. Vi ser et behov for å samle gode eksempler. En inspirasjonsside for god praktisering av jøkul - God jøkulpraksis. Foreksempel eksempler på gode feilmeldingstekster eller gode lenketekster osv. 
Kjedelig å finne på kruttet på nytt!

Ideer? Inspill?

**Comment by onordbo** [comment]
Veldig bra forslag! Dette har vi også diskutert en del i kjerneteamet, og håper på å kunne utvide portalen med designmønstre og gode eksempler av Jøkul i bruk. [Carbon Design System](https://www.carbondesignsystem.com/patterns/forms-pattern/#building-a-form) har blitt tatt opp et par ganger, kanskje vi kan ta litt inspirasjon derifra? cc @piofinn

![image](https://user-images.githubusercontent.com/4263132/137452431-0c9ca643-007e-4e8f-82a1-d00f73331ee6.png)
